modules/javafx.graphics/src/main/java/javafx/css/Selector.java

```diff
@@ -98,19 +98,22 @@ public int getOrdinal() {
     public abstract Match createMatch();
 
     /**
-     * Same as the matches method expect return true/false rather than a match.
-     * @param styleable styleable to match.
-     * @return {@code true} if this {@code Selector} applies to the given {@code Styleable}.
+     * Gets whether this {@code Selector} applies to the given {@code Styleable}.
+     * It is same as the {@link createMatch} method except it returns true/false rather than a {@code Match}.
+     * @param styleable the {@code Styleable} to match
+     * @return {@code true} if this {@code Selector} applies to the given {@code Styleable}
      */
     public abstract boolean applies(Styleable styleable);
 
     /**
-     * Same as applies, but will return pseudoclass state that it finds along the way.
-     * @param styleable styleable to match.
+     * Gets whether this {@code Selector} applies to the given {@code Styleable}.
+     * It is same as {@link applies} method except it also returns
```

Comment:
The link happens to work, but since there is more than one `applies` method, it's better to specify which one. Also, it isn't grammatically correct. Maybe something like this:

```
It is the same as the {@link applies(Styleable) applies(Styleable)} method ...
```